platform/src/main/java/org/stellar/anchor/platform/utils/TransactionHelper.java

```diff
@@ -79,11 +80,10 @@ static GetTransactionResponse toGetTransactionResponse(
         .amountIn(Amount.create(txn.getAmountIn(), amountInAsset))
         .amountOut(Amount.create(txn.getAmountOut(), amountOutAsset))
         .amountFee(Amount.create(txn.getAmountFee(), amountFeeAsset))
-        .amountExpected(Amount.create(txn.getAmountExpected(), amountExpectedAsset))
-        .customers(
-            new Customers(
-                StellarId.builder().account(txn.getToAccount()).build(),
-                StellarId.builder().account(txn.getFromAccount()).build()))
```

Comment:
Instead of removing these fields, can we just fix their values? Businesses will still need to use this info to identify their users.